students/seongbong/user/models.py

```diff
@@ -1,3 +1,11 @@
 from django.db import models
 
-# Create your models here.
+
+class User(models.Model):
+    email        = models.EmailField(max_length=50, unique=True)
+    password     = models.CharField(max_length=300)
+    phone_number = models.CharField(max_length=20, unique=True, null=True)
+    nickname     = models.CharField(max_length=20, unique=True, null=True)
+    
+    class Meta():
```

Comment:
상속받는게 없어서 소괄호는 불필요합니다.
```suggestion
    class Meta:
```

Reply:
수정했습니다. !